### ISSUE TITLE
Issue loading files

### ISSUE THREAD

**Issue by andrecastelo** [youdz/d2-stash-organizer#62]
Hey man, I'm running into several issues like this, with several characters:
![image](https://user-images.githubusercontent.com/726360/199634631-1986f909-d1ea-4b25-ba91-a57dff1be3fb.png)

Here's a zip of the characters and stash that I can't load:
[not-able-to-load.zip](https://github.com/youdz/d2-stash-organizer/files/9925194/not-able-to-load.zip)

**Comment by hunsince** [comment]
I have the same issue and I think it worked in August with the older version of the game. After the new patch the items can't be uploaded. I tested it with a few chars and everytime there is an error that an item "failed to parse". I put out everything from my inventory but afterwards it "fails" on the character weared items... Will there be an update, someone knows?

**Comment by Salepate** [comment]
Internally most of D2 item tables have changed with 2.5, the localization has also been impacted. If the original authors are still maintaining this repo I will gladly add a PR.

I have partially fixed the issue (https://github.com/Salepate/d2-stash-organizer/tree/fix/diablo2r_1.5_compat) but I still have 
some localization glitches.
the fork is deployed here: https://salepate.github.io/ (it seems to fail parsing pre 1.5 saves though so be careful with it)

**Comment by salvarubiomartinez** [comment]
> Internally most of D2 item tables have changed with 2.5, the localization has also been impacted. If the original authors are still maintaining this repo I will gladly add a PR.
> 
> I have partially fixed the issue (https://github.com/Salepate/d2-stash-organizer/tree/fix/diablo2r_1.5_compat) but I still have some localization glitches. the fork is deployed here: https://salepate.github.io/ (it seems to fail parsing pre 1.5 saves though so be careful with it)

Hi @Salepate ,
Thanks you for fixing the parsing issues.
I have used your fork and it works great, except for 'Hustle' runeword items.
Can you fix it or give me some advice on how to fix it.

Thanks

**Comment by DanilaSpevak** [comment]
> 

Hi @Salepate !

Your fork is working fine with my characters' save files. However it fails parsing my old pre-2.5 PlugY Shared Stash file saying "Unexpected header  for an item on page 20". The file opens fine in youdz's original Stash Manager.

Funny thing is that Page 20 doesn't seem to contain any weird items (which exist on some other pages, as it's a really old stash with items from different sources)

[_LOD_SharedStashSave.zip](https://github.com/youdz/d2-stash-organizer/files/12512907/_LOD_SharedStashSave.zip)

upd. Found another save file saying "Could not find base s u for unknown item". The culprit is personalized base for armor

[BadMoFo.zip](https://github.com/youdz/d2-stash-organizer/files/12513046/BadMoFo.zip)

**Comment by adm1jtg** [comment]
having lots of parsing issues as well though a just a plain enduser. NONE of my d2r characters will load I get a parsing error on every one of them.

Is the fork available for testing at some url? Nothing loads but a header from the link above

**Comment by Salepate** [comment]
Hello, I am sorry I havent replied earlier @DanilaSpevak have you managed to find out the issue? I may take a look in the upcoming days and see if I can fix that as well.

The fork can be tested at the following url @adm1jtg : https://salepate.github.io/d2v/
Cheers

**Comment by adm1jtg** [comment]
> Hello, I am sorry I havent replied earlier @DanilaSpevak have you managed to find out the issue? I may take a look in the upcoming days and see if I can fix that as well.
> 
> The fork can be tested at the following url @adm1jtg : https://salepate.github.io/d2v/ Cheers

Just tried that link with my d2r characters and I at least got most of my characters to load though my main character fails with a parse error.

Failed to parse Grand Crown at row 0, column 1: Cannot read properties of undefined (reading 'name')
and trying to pull up my plugy stash and MOST of my old lod characters I get 
Unexpected header Æ for an item on page 1
not copy pasting correctly as its showing odd characters the actual references seem to vary but the error is always an unexpected header type error. All my characters work and are also successfully imported into d2r

Also while it does show an entry for a d2r shared stash it only shows 2 items in that stash but I have NO stash tab with those 2 items in it. I will have to figure out where that is coming from

**Comment by DanilaSpevak** [comment]
> Hello, I am sorry I havent replied earlier @DanilaSpevak have you managed to find out the issue? I may take a look in the upcoming days and see if I can fix that as well.
> 
> The fork can be tested at the following url @adm1jtg : https://salepate.github.io/d2v/ Cheers

Nope. I've copied the file, removed all items from Page 20, but got same errors on Page 21.

And as for the second file, it is certainly the personified item.
